### DISCUSSION TITLE
Error - port 443: Bad access

### DISCUSSION THREAD

**Discussion by KarimovAV** [community/community#31228] [Other Feature Feedback, Questions, & Ideas]
Hello, I get an error when I work with my repository:
### _Failed to connect to github.com port 443: Bad access_

This happens with any commands when I try to send or receive something from the local repository (git clone , git push etc.):
![image](https://user-images.githubusercontent.com/71560119/187207653-d2eed839-884e-441a-bb1e-e19ebefa5c14.png)

I don't have a proxy and any antiviruses. The created repository is public:
https://github.com/KarimovAV/academyTOR.git

Tell me, please, what could be related to? What else to check?

**Comment by queenofcorgis** [comment]
Hmm usually this is proxy/firewall/antivirus related. 

Have you tried using an SSH URL and this command: 

`git remote set-url origin git@github.com:<username>/<reponame>`

**Reply by KarimovAV** [reply]
No, I will try this command. Thanks!
